### PR TITLE
Improve manage_repos task

### PR DESCRIPTION
**Pull request by elyezer**
Now manage_repos figure out the distro version instead of expecting it
to be passed as a parameter.

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -817,10 +812,8 @@ def cdn_install():
     """
     admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
 
-    os_version = distro_info()[1]
-
     # Enable some repos
-    manage_repos(os_version, True)
+    manage_repos(True)
```
Comment (omaciel) [inline automation_tools/__init__.py]: Maybe explicitly pass `cdn=True` here?

**Comment by omaciel** [comment]
ACK pending comment.

**Comment by elyezer** [comment]
Comments addressed

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**